### ISSUE TITLE
Function `label_value` does not work

### ISSUE THREAD

**Issue by Telokis** [VictoriaMetrics/VictoriaMetrics#3114]
**Describe the bug**
The function `label_value` does not work and returns no result whatsoever.

**To Reproduce**
I'm using the query `label_value(al_server_info, "fullname")` but no result show up.  
To clarify, I've tried lots of different combinations and several other metrics with the same result. I've never received any response.

**Expected behavior**
I should get a list of the label values for the given metric.

**Logs**
No log at all when performing the query.  

In Grafana's devtools, I see the query in the network tab with the query
```json
{
  "queries": [
    {
      "refId": "A",
      "expr": "label_value(al_server_info, \"fullname\")",
      "range": false,
      "editorMode": "code",
      "instant": true,
      "exemplar": false,
      "format": "table",
      "datasource": {
        "type": "prometheus"
      },
      "queryType": "timeSeriesQuery",
      "utcOffsetSec": 7200,
      "legendFormat": "",
      "interval": "",
      "datasourceId": 1,
      "intervalMs": 15000,
      "maxDataPoints": 1053
    }
  ],
  "range": {
    "from": "2022-09-15T10:27:54.244Z",
    "to": "2022-09-15T11:27:54.244Z",
    "raw": {
      "from": "now-1h",
      "to": "now"
    }
  },
  "from": "1663237674244",
  "to": "1663241274244"
}
```

And the response is
```json
{
  "results": {
    "A": {
      "frames": []
    }
  }
}
```

**Screenshots**
![image](https://user-images.githubusercontent.com/6382729/190392837-bc02d0ad-37f7-4380-85fd-1d944ec63e15.png)
![image](https://user-images.githubusercontent.com/6382729/190392898-5ff9c845-115c-4c66-9e9f-2bfb13aa2bc3.png)

**Version**
Docker image I use:
```
victoriametrics/victoria-metrics:v1.81.2
```

**Used command-line flags**
```
      - "-retentionPeriod=10y"
      - "-promscrape.config=/etc/victoriametrics/prometheus.yml"
      - "-promscrape.config.strictParse=false"
```

**Comment by laixintao** [comment]
`label_value` is not a Promql function nor MetricsQL function, it's a Grafana function which **ONLY** works at query variables (see here: https://grafana.com/docs/grafana/v9.0/datasources/prometheus/#query-variable ) Grafana uses Prometheus API of `/api/v1/label/label/values` to implement this, which Victoriametrics also support: https://docs.victoriametrics.com/#prometheus-querying-api-usage , so think it as a Grafana's function instead of Prometheus or Victoriametrics function.

So in short: it only works if you put `label_value` into a dashboard's query variables, not working when you put it into the Grafana query panes.

**Comment by hagen1778** [comment]
> So in short: it only works if you put label_value into a dashboard's query variables, not working when you put it into the Grafana query panes.

Actually, MetricsQL does have this function - see https://docs.victoriametrics.com/MetricsQL.html#label_value
But you're right in terms of using it. It only makes sense to use `label_value` in regular querying if `label` returns numeric value - see [example here](https://play-grafana.victoriametrics.com/explore?orgId=2&left=%7B%22datasource%22:%22VictoriaMetrics_demo%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22label_value(vm_app_version,%20%5C%22cluster_num%5C%22)%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D):
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/2902918/190966333-9db5d384-27dc-4a97-aa28-4d8bace6c272.png">

But if you try fetching non-numeric value it will return nothing:
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/2902918/190966410-df497990-66db-4a80-9dde-d9f0992f83ac.png">

But in general, @laixintao is correct - `label_value` function was initially Grafana's function. It has nothing to do with datasource. Prometheus does not support this function. But it was added to MetricsQL in order to support various IoT cases for users. So now it is available for regular queries.

**Comment by Telokis** [comment]
Ok, thanks for the answer!  
I know it's a Grafana function, that's how I discovered Victoria Metrics' one.  
I think the documentation should be clarified because I read it multiple times to make sure I wasn't missing anything but I would never have thought it was only working with numerical values!

I was looking for a way to bypass Grafana automatic date insertion in its `label_values` function which annoys me... (I would like all labels that were ever available even if it was 10 years ago for 30 secondes).

Thanks anyway!
